5.5/Dockerfile

```diff
@@ -24,12 +24,15 @@ EXPOSE 3306
 # to make sure of that.
 RUN rpmkeys --import file:///etc/pki/rpm-gpg/RPM-GPG-KEY-CentOS-7 && \
     yum -y --setopt=tsflags=nodocs install https://www.softwarecollections.org/en/scls/rhscl/mysql55/epel-7-x86_64/download/rhscl-mysql55-epel-7-x86_64.noarch.rpm && \
-    yum -y --setopt=tsflags=nodocs install gettext hostname mysql55 && \
+    yum -y --setopt=tsflags=nodocs install gettext hostname bind-utils mysql55 && \
     yum clean all && \
     mkdir -p /var/lib/mysql/data && chown mysql.mysql /var/lib/mysql/data && \
     test "$(id mysql)" = "uid=27(mysql) gid=27(mysql) groups=27(mysql)"
 
 COPY run-mysqld.sh /usr/local/bin/
+COPY run-mysqld-master.sh /usr/local/bin/
+COPY run-mysqld-slave.sh /usr/local/bin/
```

Comment:
@bparees I fixed it, just COPY run-*.sh /usr/local/bin so we are back in one layer.